dds/DCPS/RTPS/Sedp.h

```diff
@@ -392,51 +393,7 @@ class Sedp : public DCPS::EndpointManager<ParticipantData_t> {
   Writer participant_stateless_message_writer_;
   Writer dcps_participant_secure_writer_;
 
-  /**
-   * Special Case for the Participant Volatile Message Secure Writer, which
-   * performs key exchange. This sends the keys after the first ACKNACK from a
-   * Reader to prevent a loss of keys in the transition between authentication
-   * and key exchange. See docs/design/security.md section titled "Slow
-   * Follower Key Exchange Issue" for details.
```

Comment:
Can you remove or modify this section if it's no longer relevant?

Reply:
This is all deleted, including the RepeatOnceWriter class. Is that what you meant?

Reply:
> See docs/design/security.md section titled "Slow Follower Key Exchange Issue" for details.

I put the details in `docs/design/security.md`, so that file needs to be revised.